client/src/components/VoiceRecognition.tsx

```diff
@@ -1,4 +1,8 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import React from "react";
+import { Button } from '@material-ui/core';
```

Comment:
To keep our build time optimal we need to use one version of Material UI and make sure we're importing only the file we need rather than the whole module . Could you make this quick update please?
```suggestion
import Button  from '@mui/material/Button';
```